pages/WhatIsOppia.html

```diff
@@ -8,7 +8,7 @@
 
   <p>
     Oppia is an online learning tool that enables anyone to easily create and
-    share interactive activities ('explorations') which try to recreate the
+    share interactive activities ("explorations") which try to recreate the
```

Comment:
I think I prefer the first one better (i.e., "interactive activities" outside the brackets) since that's more understandable to new users. But I don't feel too strongly.